python/cuspatial/cuspatial/_lib/cpp/spatial_join.pxd

```diff
@@ -1,9 +1,12 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
-from libc.stdint cimport int8_t
 from cudf._lib.column cimport column_view
 from cudf._lib.table cimport table, table_view
-from cudf._lib.move cimport move, unique_ptr
+
+from libc.stdint cimport int8_t
+
+from libcpp.utility cimport move
```

Comment:
Why are we importing `move` here instead of just in the `.pyx`?

Reply:
Woops, guess this can't be a mindless find -> replace :sweat_smile: